### PR TITLE
Adding Archive folder of shows

### LINKED ISSUE #39: Archive Folder Needed
I think the archived shows should actually be put in an Archive so they don't show with all the active shows in the front page of the addon.
- JBLive.TV
- Most Recent
- Archive
- Alphabetical listing of the active shows

### PR DESCRIPTION
**Pull request by rickyphewitt**
fixes #39 

I created a show that is considered the Archive, Added an archived flag when shows are added, and updated the categories loop to include a boolean arg that checks to see if that method is called on the archive or non archive folder. I added defaults for the english strings/icon. 

Let me know if there are any changes.

Note: I wasn't sure what to put on the uri of the virtual archive folder. I just added one that would pass the length/none check toward the end of the file.

Note2: I set the english strings for the show to 25 so as to not be in conflict with the User Error show being added.

### CODE REVIEW

**Comment by rickyphewitt** [comment]
Adding photos of the new structure, tested on Arch  with kodi 16.1
addonwitharchive
archived

File: default.py [hunk 1]
```diff
@@ -491,5 +552,7 @@ def add_dir(name, url, mode, iconimage, info, page=0):
     categories()
 elif MODE == 1:
     index(NAME, URL, PAGE)
+elif MODE == 2:
+    categories(show_archived=True)
```
Comment (RobLoach) [inline default.py:556]: Slick solution around this. Was wondering how to get a sub-sub-folder.

Comment (rickyphewitt) [reply default.py:556]: Yea it was kind of daunting initially, but looking at the docs, that's the way. I could see a very complex addon having a ridiculous amount of if/elif based on Mode ha

**Comment by rickyphewitt** [comment]
Whoops! also I don't think I added the **language** to the titles/info. let me do that real quick

**Comment by MichaelTunnell** [comment]
Very nice! well done.

**Comment by RobLoach** [comment]
Merge conflict, nicely done on not having a conflict with the User Error language string. Running into an error when trying to run the Addon. Know what's up?

**Comment by rickyphewitt** [comment]
Let me take a look. I'll resolve the merge conflict and test again.

**Comment by rickyphewitt** [comment]
We can close this one in favor of: https://github.com/RobLoach/plugin.video.jupiterbroadcasting/pull/60
I rebased and tested after all the commits.

**Pull request closed without merging**